nuttx-configs/aerocore/nsh/defconfig

```diff
@@ -371,7 +348,6 @@ CONFIG_BOOT_RUNFROMFLASH=y
 #
 # Board Selection
 #
-CONFIG_ARCH_BOARD_AEROCORE=y
```

Comment:
This line needs to be restored 

and add 
CONFIG_ARCH_BOARD="aerocore"

Then if you run make menuconfig or make oldconfig -, once you have copied these back to the defconfig restore these lines:

CONFIG_ARCH_BOARD="aerocore"
CONFIG_ARCH_BOARD_AEROCORE=y